### PR TITLE
rosy-file-server: add new package

### PR DESCRIPTION
**Pull request by rosysong**
Maintainer: me
Compile tested: ar71xx, wr818, master
Run tested: ar71xx, wr818, master

@hnyman  @jow- 

Description:
This is the configuration management implementation for luci-app-rosy-file-server
https://github.com/openwrt/luci/pull/2404

### CODE REVIEW

File: net/rosy-file-server/Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rosy-file-server
+  SUBMENU:=File Transfer
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Rosy File Server over HTTP
```
Comment (neheb) [inline net/rosy-file-server/Makefile]: I recommend adding PKGARCH:=all

Comment (rosysong) [reply net/rosy-file-server/Makefile]: done, thanks!

**Comment by neheb** [comment]
I trust you'll take good care of this.

**Comment by rosysong** [comment]
@neheb  Thanks for your trust, and here is the revelant luci-apps  https://github.com/openwrt/luci/pull/2404

**Pull request merged**